lib/cloud/providers/setup_local.rb

```diff
@@ -0,0 +1,84 @@
+# Not a part of the system's functionality, just a developer utilty - written in a rush
```

Comment:
Whilst this might have been useful getting your environment in place for testing, I'm not sure it's at the standard for inclusion long-term, or that it will be useful to others in future. If it's going to stay here it ought to be uncommented and tidied up.